community/sdhcp/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+make install DESTDIR="$1" PREFIX=/usr MANPREFIX=/man
```

Comment:
The `Makefile` (installs bins to sbin)[https://git.2f30.org/sdhcp/file/Makefile.html#l46] by default so you'll either have to alter the `Makefile` or move the binaries in the build script as they should always be in `/usr/bin`.